build/packer/pmm2.el9.docker.json

```diff
@@ -0,0 +1,184 @@
+{
+  "variables": {
+    "single_disk": "false",
+    "pmm2_server_repo": "testing",
+    "pmm_client_repos": "original testing",
+    "pmm_client_repo_name": "percona-testing-x86_64",
+    "pmm_server_image_name": "perconalab/pmm-server:dev-latest"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "ami_name": "PMM2 Server [{{isotime \"2006-01-02 1504\"}}]",
+      "instance_type": "c4.xlarge",
+      "launch_block_device_mappings": [
+        {
+          "delete_on_termination": true,
+          "device_name": "/dev/sda1",
+          "volume_size": 10,
+          "volume_type": "gp2"
```

Comment:
```suggestion
          "volume_type": "gp3"
```

Reply:
It's big time to use `gp3` for it is cheaper by 40% and it can handle 30% more iops.